bower.json

```diff
@@ -34,7 +34,7 @@
   "dependencies": {
     "highcharts": "6.1.4",
     "polymer": "^2.0.0",
-    "dompurify": "2.0.5",
+    "DOMPurify": "2.0.5",
```

Comment:
I was wondering why I picked that version (it's not even the latest in the 2.0 minor). MSIE support was dropped in 3.0.0, so it should be safe to go to the latest 2.X version, 2.5.2.

Reply:
`package-lock-p3.json` would need to be updated too now